### DISCUSSION TITLE
Title not shown on react-native-screens/native-stack's modals with custom right button on android

### DISCUSSION THREAD

**Discussion by oliveirasWell** [software-mansion/react-native-screens#926] [Q&A] [answered]
# Description

I have a question about the behavior of modals on android

At this [project](https://github.com/oliveirasWell/react-screens-modal-problem) there is a problem with modals in stacks created with `createNativeStackNavigator`.

These modals are not showing the title on the header on android devices, but when you create a stack with `react-nativation/stack` the title is displayed right.

Is that behavior right? 

# Evidence

## First screen
![Screenshot_1620663303](https://user-images.githubusercontent.com/13900948/117698666-54c1d080-b19a-11eb-8d39-d594ed9309b9.png)

## Modal with react-nativation/stack

![Screenshot_1620663310](https://user-images.githubusercontent.com/13900948/117698746-686d3700-b19a-11eb-86df-7d5a99c5c45f.png)
 
## Modal with react-native-screens/native-stack modals

![Screenshot_1620663315](https://user-images.githubusercontent.com/13900948/117698769-6efbae80-b19a-11eb-9f76-3d33cf97d167.png)

**Comment by WoLewicki** [comment, marked as answer]
Did you mean the left header button? Also isn't it the same on non-modal screens on Android? Unfortunately hiding the title when left header button specified is the behavior of native header on Android and I am not sure if there is something to be done about it. You can use `headerCenter` to provide the title of your screen when using `headerLeft` at the same time.
